src/rcamera.h

```diff
@@ -44,15 +45,26 @@
 #ifndef RCAMERA_H
 #define RCAMERA_H
 
+// The only dependency // TODO review standalone mode
```

Comment:
I'm seeing that the dependency on `raymath` is really tight now... that could be a problem and something to review carefully...

Reply:
Right, we agreed to start out depending on `raymath` and I think the code is much more readable this way.
Further, without the code duplication any bug fix in `raymath` also benefits the camera module.

Alternatively, we can duplicate the relevant functions and become independent again - at the cost of duplication though.